classes/table/accessaudit_table.php

```diff
@@ -157,6 +159,14 @@ protected function get_sql_and_params($count = false) {
              LEFT JOIN {role} r ON ra.roleid = r.id";
         $sql .= " WHERE $where";
```

Comment:
I think this WHERE should be on the outside and apply to both selects, add ( ) around the selects so its really clear what the UNION is doing

Reply:
Hi @brendanheywood  This is addressed in commit 4de144be78a96d392ba0af40772a179037e29d49